### DISCUSSION TITLE
Recovering corrupted backups

### DISCUSSION THREAD

**Discussion by MaxiHuHe04** [MaxiHuHe04/iTunes-Backup-Explorer#21] [General]
(from #10)
>This tool was my last hope to restore my iPhone backup, which was failing to restore on my iPhone saying corrupted backup. (If anyone has solution to that too please let me know.)

@ankit-vs could you post the exact error message here? Does it appear on the computer (then please make a screenshot) or on the phone?

**Comment by ankit-vs** [comment]
This is not an issue with iTunes-Backup-Explorer , the issue is with the iphone back-up itself , all the back-up i took can not be recovered through Apples official methods of recovering backu-up. hence i turned to iTunes-Backup-Explorer, and it worked.
Hence, This Issue can be closed.

**Comment by MaxiHuHe04** [comment]
I know. But you asked

> (If anyone has solution to that too please let me know.)

And maybe I have a solution to make the backup restore onto a device again using the official methods. It is possible that a system file of the backup was somehow corrupted. If this is the case, the program could find it after I add that feature and you could fix it.

I would just need the error message and where it was displayed if it is still relevant.
